### ISSUE TITLE
Support extracting title for feed.

### ISSUE THREAD

**Issue by kevincox** [wezm/feedfinder#9]
For example given a feed such as:

```html
<link href=feed.atom rel=alternate title="Blog Posts" type=application/atom+xml>
```

It would be good to have the ability to get the title. This is often used when pages have multiple feeds for example a Posts Feed and a Comments Feed.

**Comment by wezm** [comment]
So something like a `title` method on the `Feed` struct that returned `Option<&str>` with the title?

**Comment by kevincox** [comment]
That would be perfect.

**Comment by wezm** [comment]
I've pushed basic support for the title to `master` if you wanted to try it out.

**Comment by kevincox** [comment]
Thanks, works perfectly.

**Comment by wezm** [comment]
Great, I've published version 0.4.0
